tests/unit/lms/resources/lti_launch_test.py

```diff
@@ -44,13 +46,13 @@ def test_it_returns_the_js_config(self, pyramid_request, JSConfig):
         assert js_config == JSConfig.return_value
 
 
-class TestCourseExtra:
+class TestNewCourseExtra:
```

Comment:
I'm not a big fan of the split up style of testing (a class per method) as it tends to string the test file out more. It's "good" when tests get very long, but I'd say that's a good time to think about refactoring the code under test. I don't think we're at that level with this code, but I suspect we used to be, which is why it's this way (we've removed a lot of this file over time).

I realise it's more of a refactor, but I'm not sure this is complicated enough to make testing it using a patch over the private method is worth it? I think it might be easier to grok (and refactor) if the testing was all done in `TestCourse`.

This is as you found it, but arguably this should always have been `TestCourse` rather than `TestCourseExtra`?

Reply:
Moving to one class with multiple methods was pretty trivial to change, done that.